autopush-common/src/db/bigtable/bigtable_client/mod.rs

```diff
@@ -849,6 +906,11 @@ impl DbClient for BigTableClientImpl {
             ));
         };
 
+        if self.refresh_router(&user.uaid).await? {
```

Comment:
I'm also fine with making the parameter `&mut user` and changing the `refreshed_at` value, but I think this is a bit clearer about what is happening.